third_party/ti_simplelink_sdk/ti_simplelink_sdk.gni

```diff
@@ -380,18 +380,18 @@ template("ti_simplelink_sdk") {
 
     if (ti_simplelink_device_family == "cc13x2x7_26x2x7") {
       libs += [
-        "${ti_simplelink_sdk_root}/source/ti/dmm/lib/gcc/m4f/libdmmlib_freertos.a",
-        "${ti_simplelink_sdk_root}/source/ti/ble5stack/libraries/cc26x2r1/OneLib.a",
-        "${ti_simplelink_sdk_root}/source/ti/ble5stack/libraries/cc26x2r1/StackWrapper.a",
-        "${ti_simplelink_sdk_root}/source/ti/ble5stack/libraries/cc26x2r1/ble_r2.symbols",
+        rebase_path(ti_simplelink_sdk_root + "/source/ti/dmm/lib/gcc/m4f/libdmmlib_freertos.a"),
```

Comment:
Why was this changed?

Please don't, this will use absolute paths which isn't desirable

"args" should be rebased to root_build_dir in the build dir

"libs" will be rebased to root_build_dir by GN so it should not be rebased in the build file